packages/store-ui/src/atoms/Slider/README.md

```diff
@@ -0,0 +1,83 @@
+# Slider
+
+## Installation
+
+From the command line in your project directory, run npm install `@vtex/store-ui` or yarn add `@vtex/store-ui`.
+
+```cmd
+npm install @vtex/store-ui
+# or
+yarn add @vtex/store-ui
+```
+
+For style, you can install our default theme, run npm install `@vtex/theme-b2c-tailwind` or yarn add `@vtex/theme-b2c-tailwind`:
+
+```cmd
+npm install @vtex/theme-b2c-tailwind
+# or
+yarn add @vtex/theme-b2c-tailwind
+```
+
+```tsx
+import { Slider } from '@vtex/store-ui'
+import '@vtex/theme-b2c-tailwind/dist/index.css'
+```
+
+## Usage
+
+```tsx
+import { Slider } from '@vtex/store-ui'
+
+function Example() {
+  return (
+    <div>
+      <Slider min={0} max={100} />
+    </div>
+  )
+}
+```
+
+## API
+
+### Component
+
+```tsx
+<Slider min={0} max={100} />
+```
+
+### CSS Selectors
+
+```css
+[data-store-slider] {
+}
+[data-store-slider-range] {
+}
+[data-store-slider-track] {
+}
+[data-store-slider-thumb='(left|right)'] {
+}
+[data-store-slider-values] {
+}
+[data-store-slider-value='(min|max)'] {
+}
+```
```

Comment:
Great addition! We should have this for all components!

Reply:
Yep! this is powerful.